spec/pagemmultiscope_spec.rb

```diff
@@ -0,0 +1,74 @@
+require File.dirname(__FILE__) + '/spec_helper'
+require File.expand_path("../../lib/pagem", __FILE__)
+
+describe PagemMultiscope do
```

Comment:
This looks like a duplicate of https://github.com/mdsol/pagem/pull/6/files#diff-cfc93d381259bb55e29c83a105475754R1